### PR TITLE
Add all plugin test

### PR DESCRIPTION
**Pull request by arugal**

### CODE REVIEW

File: gin/v3/go.sum [hunk 1]
```diff
@@ -5,6 +5,8 @@ github.com/SkyAPM/go2sky v0.6.1-0.20210314152436-fc8f3a2560f9 h1:180vs2N2/I4UhnQ
 github.com/SkyAPM/go2sky v0.6.1-0.20210314152436-fc8f3a2560f9/go.mod h1:TANzYw5EvIlTidGWvQxtvO87rM6C746HkM0xkWqnPQw=
 github.com/SkyAPM/go2sky v1.0.0 h1:kSc4EeiQRX54FBLOGhLbvMw5ks0bAccw6UuxYUZYjOg=
 github.com/SkyAPM/go2sky v1.0.0/go.mod h1:LzuySkt/TsQL8FMANZu4BX/6Rn0UfGu5KS/r9TwG/dc=
+github.com/SkyAPM/go2sky v1.1.0 h1:XT8aiCvf0qZnQz2gPfb8ROzymoog9yGNbE9wBBLny+o=
+github.com/SkyAPM/go2sky v1.1.0/go.mod h1:LzuySkt/TsQL8FMANZu4BX/6Rn0UfGu5KS/r9TwG/dc=
```
Comment (wu-sheng) [inline gin/v3/go.sum:9]: I think we need `go tiny`?

Comment (arugal) [reply gin/v3/go.sum]: Done

**Comment by arugal** [comment]
Hi @wu-sheng @mrproliu all plugin test have been added, please review :)

File: gear/test/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+version: '2.1'
+
+services:
+  mockoap:
+    image: ghcr.io/apache/skywalking-agent-test-tool/mock-collector:b01d4137fed862338d78a73b23435ccf425f9962
```
Comment (mrproliu) [inline gear/test/docker-compose.yml:21]: Could we add a base docker-compose file and put `mockoap` into this file?

**Review by mrproliu — APPROVED** [review]
Basic LGTM, please see the comment.

File: gear/test/docker-compose.yml (hunk 2 above)
Comment (arugal) [reply gear/test/docker-compose.yml:21]: Good idea

Comment (arugal) [reply gear/test/docker-compose.yml]: Done, please review :)

**Review by mrproliu — APPROVED** [review]

**Pull request merged**